app/views/sign_up/completions/show.html.erb

```diff
@@ -24,6 +24,18 @@
         sp: content_tag(:strong, decorated_session.sp_name),
       ) %>
 </p>
+<% if @multiple_factors_enabled %>
```

Comment:
Should this banner only be shown when the `select_multiple_mfa_options` feature flag is enabled?